elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -46,38 +64,56 @@ class PfePrimaryDetail extends PFElement {
 
   static get properties() {
     return {
-      // Set orientation (doesn't change)
+      // Set aria-orientation (doesn't change)
       orientation: {
         title: "Orientation",
         type: String,
         attr: "aria-orientation",
         default: "vertical",
       },
-      // Set aria role
+      // Set aria role (doesn't change)
       role: {
         type: String,
         default: "tablist",
       },
+      // The min-width of the _component_ where it has a desktop layout
+      breakpointWidth: {
+        type: Number,
+        default: 800,
+      },
+      // Read only: Displays the id of an open 'detailNav' element
```

Comment:
This is tough. If these two properties are supposed to be "read only", we probably don't want them exposed as attributes but I see that you're using them for styling. But I think its fine to leave it as is.

Reply:
I like them as an attribute so the site author can use/watch that attribute for changes if they need to tie some functionality to component state